nix/nixos-module.nix

```diff
@@ -86,6 +86,7 @@ with lib;
             description = "KMonad Instance for: " +conf-name;
             serviceConfig = {
               Type = "simple";
+              ExecStartPre = "/run/current-system/sw/bin/sleep 10";
```

Comment:
I can not test this by myself, since I am not using NixOS or even a systemd distro, but isn't a 10 seconds of sleep a lot? I think further testing to minimize this number should be done.

Reply:
For me it's just enough to fully boot and enter my password. But I'll try making the delay shorter and tell you the results

Reply:
Btw the delay has no effect on the system startup time tho since systemd starts services asynchronously, the only thing that this affects is how soon kmonad will take effect after turning on your computer

Reply:
@salahdin-ahmed I minimized the delay to 4 seconds, this is the lowest that worked for me.